### DISCUSSION TITLE
how to remove default html body?

### DISCUSSION THREAD

**Discussion by mahesh11T** [zauberzeug/nicegui#1284] [Q&A]
### Question

how can I remove this html content which loads by default?

```html
<div id="app" data-v-app=""><div class="q-layout q-layout--standard nicegui-layout" tabindex="-1" id="c0" style="min-height: 296px;"><!----><!----><div class="q-page-container" id="c1"><main class="q-page" id="c2" style="min-height: 296px;"><div id="c3" class="nicegui-content">
```

**Comment by falkoschindler** [comment]
Why would you want to do that?

**Reply by mahesh11T** [reply]
My elements are getting visible only after I deleted that content from developer html console.

I commented ui.add_head_html() element which contains .css and other things, now elements are visible, I guess there is some issue with content inside of ui.add_head_html().

**Comment by falkoschindler** [comment]
The HTML you're trying to remove is the core of NiceGUI's frontend rendering. If you remove it, there's no point in using NiceGUI.

If you have problems with some elements not showing up, please post a [minimum reproducible example](https://en.wikipedia.org/wiki/Minimal_reproducible_example) so that we can run it and investigate.

**Reply by mahesh11T** [reply]
oh I see, I am trying to debug and see where the error is coming from, will post it if not able to solve.

**Comment by mahesh11T** [comment]
The issue was in .css file which was added in ui.html_head().
